### PR TITLE
Add a "source" url to the impacts

### PR DESCRIPTION
**Pull request by magopian**
Cf https://www.notion.so/fd0068a7db8348c584cd53a3eea399f2?v=af2332265058414da4bd3aa4d381ed5f&p=da516c13667d4d098ba5d5581ca9ebae

### CODE REVIEW

**Comment by magopian** [comment]
This is what it looks like for now

In the simulator:
Capture d’écran 2022-03-17 à 14 49 02

In the explorator:
Capture d’écran 2022-03-17 à 14 48 45

File: src/Views/Impact.elm [hunk 1]
```diff
@@ -45,6 +48,14 @@ impactQuality quality =
     let
         maybeInfo =
             case quality of
+                Impact.NotFinished ->
+                    Just
+                        { cls = "btn-danger"
+                        , icon = Icon.exclamation
```
Comment (magopian) [inline src/Views/Impact.elm:54]: Maybe we'd like another icon there? Not sure how to add it to the icon pack?

File: src/Data/Impact.elm [hunk 2]
```diff
@@ -64,6 +66,7 @@ type alias PefData =
 default : Definition
 default =
     { trigram = defaultTrigram
+    , source = ""
```
Comment (n1k0) [inline src/Data/Impact.elm:69]: We may want to set default source data matching what's in the JSON here

**Comment by n1k0** [comment]
Also I suggest we add a `source` object in the JSON, having a `name` and an `url` properties:

```json
  "source": { "name": "Kering", "url": "https://…" }
```

And same in the Elm side, with a `type alias Source = { name : String, url : String }` attached to the `Definition` record.

**Review by n1k0 — APPROVED** [review]
Perfect, ship it 🚀

**Pull request merged**